### ISSUE TITLE
Parameters - Add option tag path objects with embroidery ignore

### ISSUE THREAD

**Issue by X3msnake** [lexelby/inkstitch#144]
Greetings Master

We need a way to tag a path with ignore so that that shape is ignored by the extension.

For example the helper layers will be rendered even if locked we need a way to mark parts of designs to be ignored by the extension, even if this must be tagged via XML editor

**Comment by lexelby** [comment]
Love it!  Great idea.

**Comment by lexelby** [comment]
I don't think we can use the "lock" option to indicate that a layer should be ignored, because @wwderw seems to use locking as part of his normal workflow.

Any ideas?

**Comment by X3msnake** [comment]
@lexelby 

we can start with a hidden feature like adding a embroidery_ignore tag to the layer xml object?

![image](https://user-images.githubusercontent.com/11083514/40894555-751407bc-67a2-11e8-90a1-caf0edfab437.png)

**Comment by lexelby** [comment]
Great idea!

**Comment by wwderw** [comment]
I only, at least my recollection, is only locking the instruction layer (went so far as to lock the sub layers in the objects panel) and I left them as live text to be ignored that way.

My thinking was to prevent accidental edit/deletion with the text.

Unnecessary?

**Comment by lexelby** [comment]
Probably not necessary, but I guess it can't hurt.

Thinking about it further though, I don't like the idea of using Lock to indicate "don't embroider" anyway, because it's not very obvious.
